cmis/src/main/java/nl/nn/adapterframework/extensions/cmis/mtom/MtomProxy.java

```diff
@@ -0,0 +1,74 @@
+package nl.nn.adapterframework.extensions.cmis.mtom;
+
+import java.io.IOException;
+import java.util.Map;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.context.annotation.DependsOn;
+
+import nl.nn.adapterframework.http.HttpServletBase;
+import nl.nn.adapterframework.lifecycle.DynamicRegistration;
+import nl.nn.adapterframework.lifecycle.IbisInitializer;
+import nl.nn.adapterframework.lifecycle.ServletManager;
+import nl.nn.adapterframework.util.AppConstants;
+import nl.nn.adapterframework.util.LogUtil;
+
+@IbisInitializer
+@DependsOn({"webServices10", "webServices11"})
+public class MtomProxy extends HttpServletBase implements InitializingBean, ApplicationContextAware {
```

Comment:
Kan je niet in plaats van deze proxy te bouwen, het MtomFilter voor de cmisWebServiceServlet zetten?

Reply:
Ja ik vind het ook stom, hij is nu deprecated, helaas zit hij er nog wel in; legacy probleempjes..